monitoring-custom-metrics/README.md

```diff
@@ -2,189 +2,51 @@
-
-```yaml
-spec:
-  template:
-    metadata:
-      annotations:
-        traffic.sidecar.istio.io/includeInboundPorts: "8080"
-```
-
-**All other traffic bypasses Envoy** and goes straight to the container. In Kyma, use the suggested port 8081 to expose metrics.
-
 ### Expose a sample metrics application
 
-In order to expose prometheus metrics in golang the prometheus community provides a library, [client_golang ](https://github.com/prometheus/client_golang).
-
-This example has taken a basic example where Gauge and Counter metrics are exported using the package prometheus.
-
-1. Build Docker Image
+- Deploy the application, service and servicemonitor
     ```bash
-    # (linux, mac) users
-    $ eval $(minikube docker-env)
-
-    # windows user powershell
-    > & minikube docker-env | Invoke-Expression
-
-    $ kubectl config use-context minikube
-
-    $ cd examples/monitoring-custom-metrics
-
-    $ docker build -t sample-metrics-app -f Dockerfile .
-
-    ...
-    Successfully built b41158be6124
-    Successfully tagged sample-metrics-app:latest
+    kubectl apply -f deployment -R
     ```
-[Dockerfile here](Dockerfile)
+    
+#### Access the exposed metrics in Prometheus
 
-2. Deploy sample-metrics-app
+- Run the port-forward on service core-prometheus
+    
     ```bash
-    $ kubectl apply -f k8s/sample-metrics-app.yaml
-    service "metrics-8080" created
-    service "sample-metrics-8081" created
-    deployment.extensions "sample-metrics" created
-    servicemonitor.monitoring.coreos.com "metrics" created
+    kubectl port-forward -n kyma-system svc/core-prometheus 9090:9090
     ```
+In [targets](http://localhost:9090/targets#job-sample-metrics-8081) all the **sample-metrics** endpoint will appear.
 
-    ```bash
-    $ kubectl get pods
-    NAME                             READY     STATUS    RESTARTS   AGE
-    sample-metrics-c9f998959-jd2fz   2/2       Running   0          2m
-    sample-metrics-c9f998959-kfbp8   2/2       Running   0          2m
-    sample-metrics-c9f998959-nnp2n   2/2       Running   0          2m
-    sample-metrics-c9f998959-vdnkn   2/2       Running   0          2m
-    ```
+- Use either one of the registered metrics `cpu_temperature_celsius` or `hd_errors_total` in the `expression` field in [here](http://localhost:9090/graph) and `Execute` to check the values scrapped by Prometheus.
 
-    Afterwards, run the command below to see the `istio-sidecar-injector` configmap configuration, and the annotations described above should be present.
+## Cleanup
+Run the following commands to completely remove the example and all its resources from the cluster:
 
+- Remove **label** istio-injection from **default** namespace
     ```bash
-    $ kubectl describe configmap istio-sidecar-injector -n istio-system
+    kubectl label namespace default istio-injection-
     ```
-
+- Remove **ServiceMonitor** in namespace kyma-system

```

Comment:
``Remove **ServiceMonitor** in the `kyma-system` Namespace.``

Reply:
done